### PR TITLE
Social buttons

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ScottDowne** [comment]
@Pomax just creating buttons for social.

They don't function yet.

File: src/components/call-tool.js [hunk 1]
```diff
@@ -131,7 +131,19 @@ module.exports = React.createClass({
       <section>
         <h2 className="bold">{this.context.intl.formatMessage({id: 'calling_headline'})}</h2>
         <div>{this.context.intl.formatMessage({id: 'calling_tagline'})}</div>
-        <div className="bold">SOCIAL ICONS GO HERE</div>
+        <div className="social-container">
+          <i className="fa fa-facebook" aria-hidden="true"></i>
```
Comment (Pomax) [inline src/components/call-tool.js]: Small JSX note: it's an XML syntax, not technically HTML, so any element without content can be self-closed: `<i className="fa fa-facebook" aria-hidden="true" />`, making it a tiny bit nicer.

**Comment by ScottDowne** [comment]
I borked the rebase without  noticing there were merge conflict. Fixing.

**Comment by ScottDowne** [comment]
OK, should be good now.

**Comment by Pomax** [comment]
I'll wait for the social button component, but the code as is looks reasonable to me.

**Comment by ScottDowne** [comment]
Closing this. Was having issues after you rebased and found it was easier to just drop my commit on top of master after you merged yours. Making a new branch here: https://github.com/mozilla/copyright/pull/107

**Pull request closed without merging**